README.md

```diff
@@ -69,11 +69,15 @@ This project uses `jest` and `@testing-library/react`.
 
 ## Releasing a new version to npm
 
-Your code must be reviewed and merged into `main`. A short while after this, an automated task will update versions wherever necessary and commit directly to `main`. This will appear as a commit by `ltht-epr` called "Publish"
+Your code must be reviewed and merged into `main`. A short while after this, an automated task will update versions wherever necessary and commit directly to `main`. This will appear as a commit by `ltht-epr` called "Publish".
 
 1. Switch to `main` and `git pull`
 2. Ensure versions have updated, if not wait for `ltht-epr` to update `main`
 3. Wipe your local build with `yarn clean`
 4. Install dependencies `yarn install`
 5. Build with `yarn build` or `yarn build:slow`
 6. Publish to npm with `yarn run deploy:npm`. You'll need to authenticate.
+
+## GraphQL type generation
+
+GraphQL types can be updated by running `yarn run type:gen`. This assumes that you have the appropriate GraphQL endpoint up and running. Running ehr-client in mock mode is sufficient for this purpose.
```

Comment:
Definitely not suggesting we do this now for this PR, but at some point I guess we should discuss how all this works for out mythical future customers. If they want to add any fields missing from our FHIR types do they have to submit work requests to us?

Reply:
Definitely agree with this